OmiseSDK/PaymentChooserViewController.swift

```diff
@@ -588,6 +588,9 @@ class PaymentChooserViewController: AdaptableStaticTableViewController<PaymentCh
             paymentMethodsToShow.insert(.creditCard, at: 0)
         }
 
+        if paymentMethodsToShow.contains(.truemoney) && paymentMethodsToShow.contains(.truemoneyJumpApp) {
```

Comment:
Should the ShopeePay behavior be the same? Is there any chance to refactor ShopeePay to match if so?

https://github.com/omise/omise-ios/blob/25c600db8617c9e7bc92170131f2f09cb12c501d/OmiseSDK/PaymentChooserViewController.swift#L523-L530

Reply:
I've created new ticket for that